src/libs/elektra/rand.c

```diff
@@ -53,9 +54,19 @@ void elektraRand (int32_t * seed)
 
 int32_t elektraRandGetInitSeed (void)
 {
+	int32_t initSeed = time (0);
```

Comment:
All keysets created within a second will get the same time?

Reply:
The `elektraRandGetInitSeed (...)´ is invoked by the lookup, so when two lookups are executed the same second they get the same time. Is this such bad?

Reply:
I do not know if it is a problem. gettimeofday is a cheap operation but still much more expensive than returning a constant (If everything happens within a second returning a constant would be the better solution). What I find suboptimal is that gettimeofday actually delivers micro/nanoseconds which will be discarded within glibc.

In any case we should document why it is not a problem.